docs/en/observability/configure-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,255 @@
+:aws: AWS
+
+<!-- lint disable ARNs -->
+
+<!-- lint disable arn -->
+
+<!-- lint disable yaml -->
+
+<!-- lint disable elasticsearch -->
+
+[[configure-elastic-serverless-forwarder]]
+= Configure Elastic Serverless Forwarder
+
+// add overview context i.e. what is created on deployment, what needs to be done next, contextualise and link to sample config
+
+[[lambda-permissions-policies]]
+== Permissions and policies
+
+A Lambda function's execution role is an {aws} Identity and Access Management (IAM) role that grants the function permission to access {aws} services and resources. This role is automatically created when the function is deployed and Lambda assumes this role when the function is invoked.
+
+You can view the execution role associated with your function from the **Configuration -> Permissions** section. By default this role starts with the name **serverlessrepo-elastic-se-ElasticServerlessForward-**. When the role is created, a custom policy is added to grant Lambda minimum permissions to be able to use the configured SQS queue, S3 buckets, Kinesis data stream, CloudWatch Logs log groups, Secrets manager (if using), and SQS replay queue.
+
+// check custom policy i.e. when and by whom
+
+The Lambda function is granted the following `ManagedPolicyArns` permissions. These are automatically added by default if relevant to the Events configuration:
+
+[source, bash]
+----
+arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+arn:aws:iam::aws:policy/service-role/AWSLambdaKinesisExecutionRole
+arn:aws:iam::aws:policy/service-role/AWSLambdaSQSQueueExecutionRole
+----
+
+In addition to these basic permissions, the following permissions are added when the function is created:
+// <!-- how? by whom?-->:
```

Comment:
automatically: by the cloudformation template of the forwarder

the user provides ARNs of AWS resources the Forwarder will "interact" with and the CF template generate the proper IAM role with the proper IAM policies

Reply:
Thanks, have added